Sources/CustomDump/Conformances/Foundation.swift

```diff
@@ -92,7 +92,12 @@ extension NSDate: CustomDumpRepresentable {
 
 extension NSError: CustomDumpReflectable {
   public var customDumpMirror: Mirror {
-    .init(
+    let swiftError = self as Error
+    let isNSError = swiftError._getEmbeddedNSError() === self
+    guard isNSError else {
```

Comment:
Huh! Rather than call this underscored method, is it equivalent to use another cast?
```suggestion
    let swiftError = self as Error
    let isNSError = swiftError as NSError === self
    guard isNSError else {
```

Reply:
That causes the test to fail, but it looks like we can use `type(of: swiftError) == NSError.self` to detect this case instead. I'll switch to that.